src/aws-helper/sns-sqs/README.md

```diff
@@ -0,0 +1,57 @@
+Included is a fairly simple CloudFormation template which sets up a SNS Topic configured to forward to a SQS Queue, both of which are configured with bucket policies to receive messages from your log bucket and respective topic.
```

Comment:
Consider renaming this to `s3-sns-sqs` because while CloudTrail direct notifications could use the cloudtrail queue, it has its own message format and should not be confused with the notifications that S3 generates.